lib/images.sh

```diff
@@ -13,7 +13,7 @@ elif [[ "${IMAGE_OS}" == "FCOS-ISO" ]]; then
   export IMAGE_NAME=${IMAGE_NAME:-fedora-coreos-33.20201201.2.1-live.x86_64.iso}
   export IMAGE_LOCATION=${IMAGE_LOCATION:-https://builds.coreos.fedoraproject.org/prod/streams/testing/builds/33.20201201.2.1/x86_64}
 elif [[ "${IMAGE_OS}" == "centos" ]]; then
-  export IMAGE_NAME=${IMAGE_NAME:-CENTOS_8_NODE_IMAGE_K8S_${KUBERNETES_VERSION}.qcow2}
+  export IMAGE_NAME=${IMAGE_NAME:-CENTOS_9_NODE_IMAGE_K8S_${KUBERNETES_VERSION}_test.qcow2}
```

Comment:
Is there any reason to add “_test” for image name?

Reply:
Good catch, forgot to update

Reply:
Done